metals/src/main/scala/scala/meta/internal/metals/codelenses/RunTestCodeLens.scala

```diff
@@ -47,6 +47,7 @@ final class RunTestCodeLens(
     clientConfig: ClientConfiguration,
     userConfig: () => UserConfiguration,
     trees: Trees,
+    debugProvider: DebugProvider,
```

Comment:
Not sure if the method should be in debug provider, I can move it outside and this would make less changes with moving of the providers around.